aviary/interface/methods_for_level2.py

```diff
@@ -1912,7 +1997,9 @@ def set_initial_guesses(self):
             else:
                 guesses = {}
 
-            if 'cruise' in phase_name and self.mission_method is TWO_DEGREES_OF_FREEDOM:
+            if self.mission_method is TWO_DEGREES_OF_FREEDOM and \
+                    'analytic' in self.phase_info[phase_name]["user_options"] and \
```

Comment:
I think you can replace the last 2 phrases of the 'if' with `self.phase_info[phase_name]["user_options"].get("analytic", False)`